README.md

```diff
@@ -124,9 +124,9 @@ Full abilities of VGS Collect.js and integration details you can find in our [do
 
 We strongly recommend to add the CSP to your application. Content Security Policy (CSP) is an added layer of security that helps to detect and mitigate certain types of attacks, including Cross Site Scripting (XSS) and data injection attacks. Please create CSP directives for the following domains:
 
-`connect-src https://js.verygoodvault.com https://js3.verygoodvault.com`
-`frame-src https://js.verygoodvault.com https://js3.verygoodvault.com`
-`script-src https://js.verygoodvault.com https://js3.verygoodvault.com`
+`connect-src https://js.verygoodvault.com https://js3.verygoodvault.com https://vgs-collect-keeper.apps.verygood.systems`
+`frame-src https://js.verygoodvault.com https://js3.verygoodvault.com https://vgs-collect-keeper.apps.verygood.systems`
+`script-src https://js.verygoodvault.com https://js3.verygoodvault.com https://vgs-collect-keeper.apps.verygood.systems`
```

Comment:
We need to white-list `https://vgs-collect-keeper.apps.verygood.systems` only for `connect-src` flag

Reply:
Corrected. Thanks!